programs/stake/src/rewards.rs

```diff
@@ -185,11 +185,12 @@ fn calculate_stake_rewards(
 
     let rewards = points
         .checked_mul(u128::from(point_value.rewards))
-        .unwrap()
-        .checked_div(point_value.points)
-        .unwrap();
+        .expect("Rewards intermittent calculation should fit within u128");
```

Comment:
For clarity, let's tack on the division or rename the first `rewards` variable, since the code becomes less clear if the same variable name is reused for two very different values.

and nit: does the comment mean to say "intermediate"?
```suggestion
        .expect("Rewards intermediate calculation should fit within u128")
        
        .checked_div(point_value.points)
        .unwrap(); // points_value.points is guaranteed to be non zero above
```